man/fi_endpoint.3.md

```diff
@@ -1207,6 +1208,16 @@ Operations that complete with an error increment the error counter
 and generate a completion event.  The generation of entries going to
 the CQ can then be controlled using FI_SELECTIVE_COMPLETION.
 
+As mentioned in fi_getinfo(3), the ep_attr structure can be used to
+query providers that support various endpoint attributes. fi_getinfo
+can return provider info structures that can support the minimal set
+of requirements (such that the application maintains correctness).
+However, it can also return provider info structures that exceed
+application requirements. As an example, consider an application
+requesting msg_order as FI_ORDER_NONE. The resulting output from
+getinfo may have all the ordering bits set. The application can reset
+the ordering bits it does not require before creating the endpoint.
```

Comment:
How about adding something like this to the end: "but that's optional and a provider is not required to adapt to the relaxed requirement."